docs/adr/2023-03-30-establish-consensus-based-tsc.md

```diff
@@ -0,0 +1,123 @@
+
+### Positive Consequences <!-- optional -->
+
+- Everyone has the opportunity to be heard and understood
+- Shares power between all members fairly
+- More likely to find a solution that is acceptable to all involved in resolving a given issue
+- Make sure that decisions aren't made against the will of anyone involved
+- Members are likely to assist or help in enacting the resolution
+- Builds a stronger sense of trust
+- Voting as a fall-back helps avoid eternal blocking
+
+### Negative Consequences <!-- optional -->
+
+- Consensus can be slower than voting
+- Still provides some additional powers to the facilitating Chair/s
+- May make some types of decisions harder
+- Potential for "Groupthink"
+- Requires continual active engagement
+- May make bad decisions
+- Blocking can be abused against individuals
+
+## Pros and Cons of the Options <!-- optional -->
+
+### Voting
+
+Voting with a majority rule.
+
+- Good, because it enables decisions to be made clearly and quickly
+- Good, because it gives everyone equal power
+- Bad, because it may not allow individuals to be fully heard
+- Bad, because decisions can be divisive
+
+### Unanimous Consensus
+
+Consensus where everyone must be in agreement
+
+- Good, because everyone consents to the solution
+- Bad, because it can create a fear of proposing ideas
+
+See https://www.ic.org/busting-the-myth-that-consensus-with-unanimity-is-good-for-communities/
+
+### General/rough consensus
+
+As used by the IETF
+
+- Good, because it allows progress without requiring all to agree
+- Good, because it can be faster than other consensus based methods
+- Good, because the chairperson can make judgement calls
+- Good, because it allows anonymity, encouraging people to vocalise concerns without fear of retribution or judgement
+- Bad, because it's hard to do not in-person / requires regular in-person meetings to be effective
+- Bad, because it can be subjective
+- Bad, because it puts a lot of power in the chairperson
+
+### Lazy consensus / Do-ocracy
+
+- Good, because it allows progress without requiring everyone to be involved
+- Good, because it enables faster progress
+- Good, because it can foster a sense of trust and community
+- Good, because it empowers do-ers
+- Bad, because it is possible for people to miss things
+- Bad, because it assumes silence is consent, which can result in people less likely to want to raise concerns
+- Bad, because it creates a power imbalance where people who have more time or who are more influential can push decisions through
+
+### Benevolent Dictator (for life)
+
+- Good, because it enables very fast progress
+- Good, because it can help to have a clear and consistent vision
+- Good, because there's clear accountability, which might encourage a BDFL to act in the best interests of the project
+- Bad, because it prevents diverse opinions or ideas
+- Bad, because concentrated power can be abused
```

Comment:
But... they're benevolent!  They're not going to abuse their power. 😄

Reply:
"MAY be abused" then. Not required, but allowed 🫠